go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/spf13/viper v1.7.1
 	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013 // indirect
 	google.golang.org/grpc v1.27.1
+	github.com/microsoft/moc-pkg v0.10.13-alpha.1
```

Comment:
we cannot use moc-pkg. This is a private repo and moc-sdk-for-go is a public repo

Reply:
done. moved the constants to moc-common